gcovr/__main__.py

```diff
@@ -290,7 +290,12 @@ def main(args=None):
     logger.verbose_msg("Gathered coveraged data for {} files", len(covdata))
 
     # Print reports
-    print_reports(covdata, options, logger)
+    error_occurred = print_reports(covdata, options, logger)
+    if error_occurred:
+        logger.error(
+            "Error occurred while printing reports"
+        )
+        sys.exit(1)
```

Comment:
Exit code 1 is used for command line errors. 7 is the next free one. Where are the codes documented???
```suggestion
        sys.exit(7)
```

Reply:
Hi @Spacetown. I don't think there exists a document related to the exit code currently. Maybe create an issue for it?